pages/sunos/snoop.md

```diff
@@ -0,0 +1,24 @@
+# snoop
+
+> Network packeet sniffer.
+> SunOS equivalent of tcpdump.
+
+- Capture packets on network interface e1000g0:
+
+`snoop -d e1000g0`
+
+- Capture packets to a file:
```

Comment:
```suggestion
- Capture packets and save them to a file:
```
Better wording.

Reply:
I'm not sure if this is better wording.  The use of `and` implies that the capture is both sent to the screen **and** sent to a file.

Reply:
Perhaps we could say this then:

    - Capture packets saving them in a file:

Or even:

    - Save captured packets in a file instead of displaying them: